netty/src/main/java/io/grpc/netty/AbstractNettyHandler.java

```diff
@@ -56,6 +58,9 @@
                        Http2Settings initialSettings) {
     super(decoder, encoder, initialSettings);
 
+    // Set the timeout for graceful shutdown.
+    gracefulShutdownTimeoutMillis(GRACEFUL_SHUTDOWN_TIMEOUT);
```

Comment:
The default was moved to a builder that's not being used in our case ... so it ends up being zero by default.